crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -346,6 +364,14 @@ pub enum Command {
         #[serde(skip)]
         command: Option<Box<GenerateCommand>>,
     },
+    // TODO:[telemetry] Unhide this in `1.12`
+    /// Enable or disable anonymous telemetry
+    #[clap(hide = true)]
+    Telemetry {
+        #[clap(subcommand)]
+        #[serde(flatten)]
+        command: Option<Box<TelemetryCommand>>,
```

Comment:
What's the purpose of the `Box` here?

Reply:
I this pattern copied from another command I believe, removed!